<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="camera_Button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="env_Button">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>环境信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_Button">
    <property name="geometry">
     <rect>
      <x>948</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="light">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>380</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>亮度</string>
    </property>
   </widget>
   <widget class="QLabel" name="humi">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>420</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>湿度</string>
    </property>
   </widget>
   <widget class="QLabel" name="tem">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>460</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>温度</string>
    </property>
   </widget>
   <widget class="QLabel" name="ip">
    <property name="geometry">
     <rect>
      <x>888</x>
      <y>610</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="port">
    <property name="geometry">
     <rect>
      <x>878</x>
      <y>650</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="light_text">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>380</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="hum_text">
    <property name="geometry">
     <rect>
      <x>938</x>
      <y>420</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tem_text">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>460</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_text">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>610</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_text">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>650</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2333</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_view">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>721</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string>画面</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>520</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="led_Button">
       <property name="text">
        <string>LED</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fan_Button">
       <property name="text">
        <string>风扇</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bee_Button">
       <property name="text">
        <string>蜂鸣器</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
